stdlib/universe/kaufmansAMA_test.go

```diff
@@ -0,0 +1,209 @@
+package universe_test
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/querytest"
+	"github.com/influxdata/flux/stdlib/universe"
+	"testing"
+)
+
+func TestKamaOperation_Marshaling(t *testing.T) {
+	data := []byte(`{"id":"kaufmansAMA","kind":"kaufmansAMA","spec":{"n":1}}`)
+	op := &flux.Operation{
+		ID: "kaufmansAMA",
+		Spec: &universe.KamaOpSpec{
+			N: 1,
+		},
+	}
+	querytest.OperationMarshalingTestHelper(t, data, op)
+}
+
+func TestKama_PassThrough(t *testing.T) {
+	executetest.TransformationPassThroughTestHelper(t, func(d execute.Dataset, c execute.TableBuilderCache) execute.Transformation {
+		s := universe.NewkamaTransformation(
+			d,
+			c,
+			&universe.KamaProcedureSpec{},
+		)
+		return s
+	})
+}
+
+func TestKama_Process(t *testing.T) {
+	testCases := []struct {
+		name string
+		spec *universe.KamaProcedureSpec
+		data []flux.Table
+		want []*executetest.Table
+	}{
+		{
+			name: "flux.TInt input",
+			spec: &universe.KamaProcedureSpec{
+				N:       10,
+				Columns: []string{"_value"},
+			},
+			data: []flux.Table{&executetest.Table{
```

Comment:
Some of the tests for EMA use row-wise tables:
https://github.com/influxdata/flux/blob/master/stdlib/universe/exponential_moving_average_test.go#L78

Could you add at least one test case here that does as well?

Reply:
This will force the table block size to be exactly one row, so you can verify that KAMA works right for tables that span multiple blocks.